db/migrate/20161130153206_add_last_updated_at_to_content_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastUpdatedAtToContentItems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :content_items, :last_updated_at, :datetime
```

Comment:
I would mimic the same name as in the `content_store`. This will improve code readability.
I would move from `last_updated_at` to `public_updated_at`.